deeponet/benchmark_models.py

```
@@ -0,0 +1,282 @@
+def run_dem(key):
+    print("--- Starting DEM Benchmark ---")
+    k_model, k_train = jr.split(key)
+    # Same architecture as PINN
+    model = ElasticityNetwork(hidden=64, n_layers=4, n_fourier=16, key=k_model)
+    optimizer = optax.adam(1e-3)
+    opt_state = optimizer.init(eqx.filter(model, eqx.is_array))
+    
+    @eqx.filter_jit
+    def make_step(model, opt_state, domain_pts, top_pts):
+        # E values
+        E_dom = get_E_at_points(domain_pts, None, None)
```

Ruff (0.15.2)

[error] Local variable `E_dom` is assigned to but never used

Remove assignment to unused variable `E_dom`

(F841)